site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -15,6 +15,36 @@
             </p>
             <input type="text" id="input-provide-full-path" class="full-width" name="eg_config_path" value="" placeholder="optional/subdirectory"/>
         </label>
+        <label id="upload_sections">
+            <p>
+                <strong>Do you want to restrict this course material to some sections?</strong><br />
```

Comment:
Get rid of the extra `<br />` here
You could also use <p class="black-message"> instead of a nested `<strong>` since you're just emphasizing all of the 
content under the paragraph tag.

Reply:
Changed to <p class="black-message">